### PR TITLE
Ft complete quickstart

### PR DESCRIPTION
**Pull request by MutegekiHenry**
### What does this PR do?
It adds content to sections of the Quick Start Page below:

- Email Confirmation

- Create Project

- Create Application

- Access Application

Screenshots: 

Screenshot for create project.
Screenshot from 2020-06-16 15-54-51

### CODE REVIEW

File: cranecloud/docs/quickstart/quickstart.md [hunk 1]
```diff
@@ -9,12 +9,40 @@
...
+    - Submit the form.
+    - The page will reload and display your newly created application. 
+    
+    ![](../img/createAppForm.png)
 
 ## Access application
+1. Navigate into your project to view a list of applications you created under that project.
+    ![](../img/appDashwithApps.png)
+
+2. To access a particular application, click on the URL attached to its card. The green dot on the App card symbolizes that the App is running without any issues.
```
Comment (inno-asiimwe) [inline cranecloud/docs/quickstart/quickstart.md:48]:
```suggestion
2. To access a particular application, click on the URL attached to its card. The green dot on the application card symbolizes that the App is running.  
```

File: cranecloud/docs/quickstart/quickstart.md [hunk 2]
```diff
@@ -9,12 +9,40 @@
...
+
+3. First, `select a cluster` where your project will reside. Fill in the form with `Project Name` and `Project description` and click `PROCEED` to submit.
+           ![](../img/createProjectForm.png)
+
+4. Your Page will auto reload to display the newly created project.
 
 ## Create Application
+1. To create an Application, you need click on a project card to drill into that project.  
+
+2. Then, click the Create button on the top right corner of your apps' dashboard, to display create App Form.
```
Comment (inno-asiimwe) [inline cranecloud/docs/quickstart/quickstart.md:32]:
```suggestion
2. Then, click the `Create button` on the top right corner of your apps' dashboard, to display create App Form.
```

**Comment by MutegekiHenry** [comment]
> Great work @MutegekiHenry , application and App are being used inconsistently. I suggest we use application throughout.

Okay @inno-asiimwe let me quickly resolve this

**Comment by MutegekiHenry** [comment]
@inno-asiimwe worked on the feedback

**Pull request merged**